README.md

```diff
@@ -1,8 +1,8 @@
-# action_kit_rest
+# action\_kit\_rest
```

Comment:
Why is this needed?

Reply:
This tells Markdown that these are real underscores, not an attempt to underline/italicize something. It's probably not strictly necessary, but my editor was highlighting it in red and this made it stop :slightly_smiling_face: